docs/source/tutorials_source/platform/tutorial_active_learning.py

```diff
@@ -94,39 +113,27 @@
 # -----------------
 #
 # Import the Python frameworks we need for this tutorial.
+# We need numpy, scikit-learn and lightly as non-standard packages.
```

Comment:
can you also show the commands for installation here? I think some people just want to follow instructions